app/styles/components/bc-list.scss

```diff
@@ -3,6 +3,7 @@ $image-size: 40px;
 
 .bc-list {
 	width: 100%;
+	min-height: 150px;
```

Comment:
Having min-height here will cause other issues with drop downs. Min height was removed to fix another bug. This is not a correct fix to this issue.